tasks/scripts/build-release-notes

```diff
@@ -12,8 +14,20 @@ chmod +x release-me/releaseme
   --last-commit-SHA=$LAST_COMMIT_SHA \
   --release-version=$RELEASE_VERSION \
   --ignore-authors=dependabot \
-  > built-notes/notes.md
+  > $FILE
+
+if [ -f resource-type-versions/versions.yml ]; then
+  cat >> $FILE <<EOF
+
+## 📦 Bundled resource types
+
+<details>
 
-if [ -f resource-versions/versions ]; then
-	cat resource-versions/versions >> built-notes/notes.md
+EOF
+  cat resource-type-versions/versions.yml | while read line; do
+    name=$(printf $line | cut -d':' -f1 | xargs)
+    version=$(printf $line | cut -d':' -f2 | xargs)
+    ghlink="https://github.com/concourse/$name-resource/releases/tag/$version"
+    echo "- $name: [$version]($ghlink)" >> $FILE
+  done
```

Comment:
Harness the arcane powers of `awk`:

```
awk 'BEGIN {FS = ": "}; { printf "- %s: [%s](https://github.com/concourse/%s/releases/tag/%s)\n", $1, $2, $1, $2 }' resource-type-versions/versions.yml
# or slightly more readable:
awk 'BEGIN {FS = ": "}; { url = sprintf("https://github.com/concourse/%s/releases/tag/%s", $1, $2); printf "- %s: [%s](%s)\n", $1, $2, url }' resource-type-versions/versions.yml
```

Reply:
nice, haven't really bothered to learn `awk` effectively yet but seems pretty useful - cleaned up in 1749c3d1e0dd103f198ecf9cfae847ac6100c4f3